haystack/nodes/prompt/prompt_node.py

```diff
@@ -293,66 +476,95 @@ def get_predefined_prompt_templates() -> List[PromptTemplate]:
     return [
         PromptTemplate(
             name="question-answering",
-            prompt_text="Given the context please answer the question. Context: $documents; Question: "
-            "$questions; Answer:",
+            prompt_text="Given the context please answer the question. Context: {join(documents)}; Question: "
```

Comment:
Changed the template to return answers. So it's the standard QA template

Reply:
This is a breaking change then, right? Definitely one we can and should do, but we should have a quick note in the release notes giving "migration help". I guess it's just listing two options: 1) If you want to keep the string output from your PromptNode: use an alternative template without the "join" OR 2) If you used output shaper before in your pipeline => drop it.